src/style.css

```diff
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search-input {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  padding: 26px 0 24px 62px;
+  background-color: #fff;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: center left 26px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-input::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-input:focus {
+  border-radius: 3px;
+  font-weight: 900;
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background: url(./images/Search.svg) no-repeat 25px center, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
I believe it would be enough to do so
```
  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```
so that we just add another sub-property and do not overwrite our icon